sessionctx/binloginfo/binloginfo.go

```diff
@@ -14,53 +14,57 @@
 package binloginfo
 
 import (
+	"io/ioutil"
 	"regexp"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/pingcap/parser/terror"
+	"github.com/pingcap/tidb-tools/tidb-binlog/node"
+	pumpcli "github.com/pingcap/tidb-tools/tidb-binlog/pump_client"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/metrics"
 	"github.com/pingcap/tidb/sessionctx"
 	binlog "github.com/pingcap/tipb/go-binlog"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
-	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
 func init() {
 	grpc.EnableTracing = false
+	// don't need output pumps client's log
+	pumpcli.Logger.Out = ioutil.Discard
 }
 
 var binlogWriteTimeout = 15 * time.Second
```

Comment:
Could we remove this variable and the function of `SetGRPCTimeout`? It seems we don't use them anymore.